tlslite/tlsrecordlayer.py

```diff
@@ -738,6 +756,8 @@ def _getMsg(self, expectedType, secondaryType=None, constructorType=None):
                 yield Alert().parse(p)
             elif recordHeader.type == ContentType.application_data:
                 yield ApplicationData().parse(p)
+            elif recordHeader.type == ContentType.heartbeat:
+                yield Heartbeat().parse(p)
```

Comment:
if _getMsg handles the messages itself (line 732), why return them to calling application?

to be precise: I have no problem with either option, but not both at the same time...